graphql/e2e/directives/docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
       --trace "ratio=1.0;"
 
   lambda:
-    image: dgraph/dgraph-lambda:latest
+    image: dgraph/dgraph-lambda:latest # Need to push arm lambda image to dockerhub and then uncomment
```

Comment:
Any push would clobber the amd with the same tag. We need to use the `docker buildx` extensions. See this issue for some context: https://github.com/dgraph-io/dgraph-lambda/issues/36

Reply:
Yes I have done the buildx method on a forked repo according to https://github.com/dgraph-io/dgraph-lambda/issues/36. Just waiting on the go ahead to push the image for arm/amd platforms

Reply:
Resolving this .. as it's done based on this https://github.com/dgraph-io/dgraph-lambda/issues/36